data/MI/programs.json

```diff
@@ -109,22 +109,22 @@
     "authority": "mi-indiana-michigan-power",
     "authority_type": "utility"
   },
-  "mi_lansingBoardOfWaterAndLight_lansingBWLElectrificationPrograms": {
+  "mi_lansingBoardOfWaterAndLight_lansingBWLENERGYSTARApplianceRebates": {
```

Comment:
This churn is just because HERO sorts case-sensitively. That was easier to implement in HERO than case-insensitive sorting would have been, and this churn is a one-time cost.